README.md

```diff
@@ -1,7 +1,7 @@
 Two player chess made by Touro students
 
 The Chess board of the program should look like this:
-![chess board](https://cutechess.com/cutechess.png)
+![chess board](   )
```

Comment:
Do not remove this line

Reply:
I fixed this in my README branch and pushed that change. I did not pull that update into the undoSupport branch; due to unresolved merge conflicts, I can't fetch upstream.